cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -251,47 +265,53 @@ func (a *StaticAutoscaler) RunOnce(currentTime time.Time) errors.AutoscalerError
 	unschedulablePodsToHelp = a.filterOutYoungPods(unschedulablePodsToHelp, currentTime)
 
 	if len(unschedulablePodsToHelp) == 0 {
+		scaleUpStatus.Result = status.ScaleUpNoUnschedulablePods
 		glog.V(1).Info("No unschedulable pods")
 	} else if a.MaxNodesTotal > 0 && len(readyNodes) >= a.MaxNodesTotal {
+		scaleUpStatus.Result = status.ScaleUpLimitExceeded
```

Comment:
This is inconsistent with what happens when we hit limits for each node group's size. Can we use "no options" result here as well?

Reply:
done